docker/public/wovn.json

```diff
@@ -0,0 +1,11 @@
+{
+    "project_token": "TOKEN",
```

Comment:
I just thought it would be nicer if we could `sed` the TOKEN with provided token via `--build-arg`.

Reply:
interesting idea. Could be useful but most of the time you'll change more than just token and it would be awkward to specify everything via build args imo